torch/_inductor/ir.py

```diff
@@ -3225,6 +3225,8 @@ def __init__(
                 if V.graph.cpp_wrapper
                 else f"aten.{kernel.__name__}"
             )
+        elif getattr(torch.ops, kernel.__name__, None) is kernel:
+            self.kernel = f"torch.ops.{kernel.__name__}"
```

Comment:
Does this hardcode torch.ops.run_and_save_rng_state ? (can we avoid this?)

Reply:
Ah good catch. Fixed it now.